.mockery.yaml

```diff
@@ -34,6 +34,14 @@ packages:
   github.com/Dynatrace/dynatrace-operator/pkg/controllers:
     interfaces:
       Reconciler:
+  github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube:
```

Comment:
I have no strong preference where the mocks should be, I might lean towards "next to the interface it mocks".

But we should be consistent (and the team should agree on 1 way).

Do we have a ticket to move the mocks similar to how you did it now?
If not I would just follow the current pattern, and discuss it separately if we want to move away from it. wdyt?

Reply:
agree - we need to define how we want to use local mocks/ and mocks we reuse between pakcages.
I prefer to document and discuss with team and later we will not question it again.